### PR TITLE
Project examples with Spring5

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Comment by diegolovison** [comment]
@karesti

File: spring5-session/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    <name>JBoss JDG Quickstart: Spring5 Session</name>
+
+    <properties>
+        <failOnMissingWebXml>false</failOnMissingWebXml>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <version.org.infinispan>9.4.6.Final</version.org.infinispan>
+        <version.spring-boot>1.5.16.RELEASE</version.spring-boot>
```
Comment (karesti) [inline spring5-session/pom.xml]: 2.1.1.RELEASE

Comment (karesti) [reply spring5-session/pom.xml]: Spring5 + SB 2.x

**Review by karesti — CHANGES_REQUESTED** [review]

File: spring5-session/pom.xml (hunk 1 above)
Comment (diegolovison) [reply spring5-session/pom.xml]: Done

**Comment by karesti** [comment]
everything works fine ?

**Comment by diegolovison** [comment]
Yes. I also did some updates in the source code because some classes were removed in Spring Boot. LGTM.

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
@maybe we should upgrade to the datagrid version (in another commit)

**Comment by karesti** [comment]
the 9.4.6-Final-redhat-00001.jar

**Comment by karesti** [comment]
@diegolovison not sure this is the version so far, I think it is the one I posted just before

**Comment by diegolovison** [comment]
@karesti IMO, we should keep jboss-jdg-quickstarts with upstream ISPN version. If we would like to test another version we can `mvn -Dversion.org.infinispan=9.4.6-Final-redhat-00001.jar`

**Pull request merged**